pkg/controller/pravega/bookie.go

```diff
@@ -191,6 +191,10 @@ func MakeBookieConfigMap(pravegaCluster *v1alpha1.PravegaCluster) *corev1.Config
 		configData["BK_AUTORECOVERY"] = "true"
 	}
 
+	for k, v := range pravegaCluster.Spec.Bookkeeper.Options {
+		configData[k] = v
```

Comment:
@Tristan1900 is this working to you? I thought you had to add the `BK_` prefix to the options.